patchwork-fml/src/main/java/net/minecraftforge/fml/ModContainer.java

```diff
@@ -46,6 +54,21 @@ public final String getNamespace() {
 		return namespace;
 	}
 
+	@SuppressWarnings("unchecked")
+	public <T> Optional<T> getCustomExtension(ExtensionPoint<T> point) {
+		return Optional.ofNullable((T) extensionPoints.getOrDefault(point, () -> null).get());
```

Comment:
> getOrDefault(x () -> null)

Isn't that the same thing is `get()`? Maybe this is some quirk of IdentityHashMap but I don't think so.

Reply:
Maybe Forge was trying to provide some fallback so they leave `() -> null` here.

Actually I'm not too sure either, perhaps we can leave it there to keep the behavior consistent with Forge.

Reply:
Hang on, i see what's going on. It's returning some supplier, and this is more concise because it avoids having to check before unboxing.